CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ IMPROVEMENTS:
    an idempotent operation. [GH-1903]
  * credential/approle: At least one constraint is required to be enabled while
    creating and updating a role [GH-1882]
+ * secret/cassandra: Added consistency level for use with roles [GH-1931]
```

Comment:
Yeah, this is a Mitchell thing that we have kept. He really really dislikes Changelog changes in PRs.